rotkehlchen/exchanges/bitstamp.py

```diff
@@ -261,10 +261,9 @@ def query_online_deposits_withdrawals(
         )
 
         # also query crypto transactions, to get address and transaction id (but no fee)
-        # TODO: The used query range should be replaced by cache for this: https://github.com/rotki/rotki/issues/5684  # noqa: E501
         crypto_last_query_name = f'{self.location}_{self.name}_last_cryptotx_offset'
         with self.db.conn.read_ctx() as cursor:
-            cursor.execute('SELECT end_ts FROM used_query_ranges WHERE name=?', (crypto_last_query_name,))  # noqa: E501
+            cursor.execute('SELECT value FROM key_value_cache WHERE name=?', (crypto_last_query_name,))  # noqa: E501
```

Comment:
no need to do it, this is just question, but why not use the cache function you made here? More work / less optimized?

Reply:
I did not notice because before this change also it was doing a direct query. But yeah, it will take an extra function call if we use it here.